packages/dropdownMenu/stories/Dropdown.stories.tsx

```diff
@@ -41,8 +72,13 @@ export const Default = () => (
   </DropdownMenu>
 );
 
-export const WithDangerousAction = () => (
-  <DropdownMenu trigger={<PrimaryDropdownButton>Menu</PrimaryDropdownButton>}>
+export const Default = Template.bind({});
```

Comment:
❓ - I am getting some accessibility errors. Is this a priority for us? 
![image](https://user-images.githubusercontent.com/7118459/144924550-25d89386-94a5-4d21-96b8-0d519a07c42b.png)

Reply:
Ah, good catch, this is pre-existing. I will log a separate ticket to fix the a11y.